guides/shop-managers-guide/77-configuring-address-validation.md

```diff
@@ -0,0 +1,12 @@
+# Address validation
+
+You can find the settings for the for the Address Validation under Settings -> Address Validation in the Admin.
+![Address validation](_assets/77-address-validation-settings.png)
+
+In the settings you can see a table which displays the list of address validation services that have been set up and the country to which it applies.
+
+The Add Address Validation Services button provides a modal to add more services to the list.
+![Add addres validation service](_assets/77-add-address-validation-service.png)
+
+- Validation service: This dropdown shows the list of address validation services that have been set up with the appication. It comes with a default Test Validation for testing.
+- Country filter: This dropdown displays a list of countries to which the address validation should apply.
```

Comment:
- **Country filter**: The countries to which the address validation should apply.